MekHQ/src/mekhq/gui/CampaignGUI.java

```diff
@@ -2821,8 +2821,11 @@ private void refreshFunds() {
 
     private void refreshRating() {
         if (getCampaign().getCampaignOptions().useDragoonRating()) {
-            String text = "<html><b>Dragoons Rating:</b> "
-                    + getCampaign().getUnitRating() + "</html>";
+        	// this is the one situation where we do want to refresh the rating,
+        	// as it means something has happened to influence it
+        	getCampaign().getUnitRating().reInitialize();
+        	
+            String text = "<html><b>Dragoons Rating:</b> " + getCampaign().getUnitRatingText() + "</html>";
             lblRating.setText(text);
         } else {
             lblRating.setText("");
```

Comment:
I believe that when this method was written, FMMr was the only options for a unit rating, which is why the rating label only ever gets a value when that is selected.  It seems that, now that we have COR as an alternative, we probably should put in an _else if_ condition to populate the label with it's unitRatingText value.